### DISCUSSION TITLE
How do I disable path completion on remote hosts?

### DISCUSSION THREAD

**Discussion by weilbith** [marlonrichert/zsh-autocomplete#352] [Q&A] [answered]
## What do you want?
Allow to disable the completion of paths while typing commands like `scp` with paths on other hosts.

## Why do you want this?
I use a YubiKey to authenticate myself to other hosts. When I start to type a path for a `scp` command the auto-completion continuously unfocuses my terminal  and focus a window to authenticate myself. Clicking it away will just cause a loop each time I type something. This is incredibly annoying. Even when I authenticate myself, due to my strict caching settings it will require me to authenticate over and over again.

## Who else would benefit from this?
I guess everyone using the `scp` command and a password protected private SSH key or other authentication mechanisms that require user interactions.

## Example scenario

`scp ~/local/path user@remote_name:/p...` <-- immediate trouble

**Comment by marlonrichert** [comment, marked as answer]
Use this:
```zsh
zstyle ':completion:*' remote-access no
```
See https://zsh.sourceforge.io/Doc/Release/Completion-System.html#index-remote_002daccess_002c-completion-style

**Reply by weilbith** [reply]
Okay it is that easy. Sorry that I thought I must disable for this plugin here.
Thanks for the help! Saves me a lot of nerves.

**Reply by marlonrichert** [reply]
You're welcome!

PS: If you enjoy using my software, please consider [sponsoring me](https://github.com/sponsors/marlonrichert). 🙂
